goolib/goospec.go

```diff
@@ -403,6 +404,15 @@ func (ps *PkgSpec) normalize() {
 		}
 		*str = filepath.Clean("/" + *str)[1:]
 	}
+	if runtime.GOOS == "windows" {
```

Comment:
I don't think this is needed and in fact is kinda pointless, any filesystem actions taken by googet will run through oswrap library, and that will call filepath.Clean on all paths. That will replace all "/" with "\" on windows

Reply:
You're right that this change is irrelevant for filesystem actions. This was done for backward compatibility. These strings are compared verbatim with the result of `mapFiles()` in `verifyFiles()` in `goopack.go`. Without this change, old `goospec` files with backslashes would stop working. This is just to ensure that internally, all package file paths are represented with forward slashes. Does this help the change make sense?

Reply:
Ah I see the code path where goopack will use this, I don't think this would have any other side effects either because of the call to clean